Code/Tools/ProjectManager/Source/FormOptionsWidget.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QWidget>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QCheckBox)
+QT_FORWARD_DECLARE_CLASS(QFrame)
+
+namespace AzQtComponents
+{
+    class CheckBox;
+}
+
+namespace O3DE::ProjectManager
+{
+    class FormOptionsWidget : public QWidget
+    {
+        Q_OBJECT
+
+    public:
+        FormOptionsWidget(const QString& labelText,
+                          const QStringList& options,
+                          const QString& allOptionsText,
+                          const int optionItemSpacing = 24,
```

Comment:
normally we do not mark pass-by-value parameters as const - are you intending to signal some intent about how it is used?
```suggestion
                          int optionItemSpacing = 24,
```

Reply:
Main intent is that the spacing value does not change over the course of the function. If that is not necessary I can remove this.